XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -214,4 +218,12 @@
   <ItemGroup>
     <Folder Include="Views\Snackbar\Helpers\WPF\" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Converters\StringToIntConverter.shared.cs">
+      <SubType></SubType>
+    </Compile>
+    <Compile Update="Converters\StringToDoubleConverter.shared.cs">
+      <SubType></SubType>
+    </Compile>
+  </ItemGroup>
```

Comment:
I don't think this is needed, they will be added automatically. I know this happened from the IDE, but let's revert it to keep the csproj clean